packages/api-plugin-fulfillment/src/schemas/schema.graphql

```diff
@@ -0,0 +1,414 @@
+"Allowed fulfillment types, extended by fulfillment-type plugins"
+enum FulfillmentType {
+  "Default fulfillment type when none is decided by user"
+  undecided
+}
+
+"Default empty object for additionalData"
+type emptyData {
```

Comment:
We are using PascalCase for all types and inputs in the GraphQL schema files across the project. I think we should do too here.

Reply:
Updated